sql/core/src/test/scala/org/apache/spark/sql/execution/GlobalTempViewSuite.scala

```diff
@@ -140,8 +140,8 @@ class GlobalTempViewSuite extends QueryTest with SharedSQLContext {
 
       assert(spark.catalog.listTables(globalTempDB).collect().toSeq.map(_.name) == Seq("v1", "v2"))
     } finally {
-      spark.catalog.dropTempView("v1")
-      spark.catalog.dropGlobalTempView("v2")
+      spark.catalog.dropGlobalTempView("v1")
+      spark.catalog.dropTempView("v2")
```

Comment:
Hi, @xubo245 .
Could you split the bug and new improvement into two separate PR?
Then, your PR will get reviews more easily.

Reply:
Ok, done. Please review: https://github.com/apache/spark/pull/20250